app/models/admin_auction_status_presenter_factory.rb

```diff
@@ -46,6 +46,8 @@ def other_purchase_card_presenter
       AdminAuctionStatusPresenter::Future
     elsif future? && auction.unpublished?
       AdminAuctionStatusPresenter::ReadyToPublish
+    elsif available?
```

Comment:
seems like we are now checking for available twice in this conditional -- thoughts on whether we should just check once?